apps/schools/models/students.py

```diff
@@ -41,13 +44,13 @@ class StudentGroup(models.Model):
     section = models.CharField(
         max_length=10,
         choices=Alpha_list,
-        blank=True,
+        blank=True, #Shivangi Check if this should be there!
         null=True,
-        default=''
+        default='' #Shivangi, should there be No Section? Or just null?
```

Comment:
We can have a default of 'No Section' , null as false and remove ' ' from the alpa list